src/engine/test/source/router/store_test/internal/router_table/0

```diff
@@ -0,0 +1 @@
+[{"name":"allow_all_C3","priority":1,"filter":"filter/allow_all_C3/0","target":"environment/env_C3/0"},{"name":"allow_all_A1","priority":50,"filter":"filter/allow_all_A1/0","target":"environment/env_A1/0"},{"name":"allow_all_B2","priority":202,"filter":"filter/allow_all_B2/0","target":"environment/env_B2/0"}]
```

Comment:
A newline is missing

Reply:
This is ok, this file will be regenerated without end of line